xt/discordable.t

```diff
@@ -0,0 +1,53 @@
+#!/usr/bin/env perl6
+BEGIN %*ENV<PERL6_TEST_DIE_ON_FAIL> = 1;
+%*ENV<TESTABLE> = 1;
+
+use lib <lib xt/lib>;
+use Test;
+use Testable;
+
+my $t = Testable.new: bot => ‘Evalable’, :discord;
+
+is $t.our-nick, ‘testable’, ‘our-nick is testable...’;
+is $t.^attributes.first(‘$!our-nick’).get_value($t), ‘discord6’, ‘... but we're faking discord6’;
+
+$t.common-tests: help => “Like this: {$t.bot-nick}: say ‘hello’; say ‘world’”;
+
+$t.shortcut-tests: <e: e6: eval: eval6:>,
+                   <e e, e6 e6, eval eval, eval6 eval6,>;# what what, what:>;
```

Comment:
Note that I had to remove `what:` from the disallowed shortcut array. If I leave it inside, the `discordable.t` fails:
```
ok 35 - “eval6” shortcut does not work               
ok 36 - “eval6,” shortcut does not work                                                                   
ok 37 - “what:” shortcut does not work                                                                    
not ok 38 - basic “nick:” query       
# Failed test 'basic “nick:” query'                  
# at /home/tab/src/perl6/taboege-whateverable/xt/lib/Testable.pm6 (Testable) line 133
# expected: [/^ <me($t)>‘, rakudo-moar ’<sha>‘: OUTPUT: «hello␤»’ $/]                                     
#  matcher: 'infix:<~~>'                                                                                  
#      got: ["testable, https://github.com/perl6/whateverable"]
# Test failed. Stopping test suite, because PERL6_TEST_DIE_ON_FAIL environmental variable is set to a true value.
# Bot process finished (exit code=0, signal=1)
```
Seems like the response to the `what: url` is received only in the next test. But there should be no response at all.

Reply:
https://github.com/perl6/whateverable/blob/f1563a651f351c7ae8528268d0d257708aa065e6/lib/Whateverable.pm6#L48

It feels like `$msg.args[1].starts-with: ‘what:’` is false, I guess Discordable doesn't fake args enough? It needs to be fixed, because otherwise evalable will respond to `what: ` messages from discord.

Reply:
Success :-)